docs/release-notes/8.12.asciidoc

```diff
@@ -1,6 +1,23 @@
 [[release-notes-header-8.12.0]]
 == 8.12
 
+[discrete]
+[[release-notes-8.12.1]]
+=== 8.12.1
+
+[discrete]
+[[bug-fixes-8.12.1]]
+==== Bug fixes
+* Fixes misaligned elements in the top navigation bar ({pull}175516[#175516]).
+* Fixes a bug that affected search results when you entered an agent name that included a dash (`-`) ({pull}175134[#175134]).
+* Fixes the UI bug that hid frequency configuration options for rule actions when you created or edited a rule ({pull}175050[#175050]).
+* Removes the option to select a Data view when modifying a rule's filter ({pull}174922[#174922]).
```

Comment:
```suggestion
* Removes the option to select a data view when modifying a rule's filter ({pull}174922[#174922]).
```

Reply:
I was actually wondering about this because we capitalize "data" in our data view [docs](https://www.elastic.co/guide/en/security/master/data-views-in-sec.html) and I can't recall if we discussed the capitalization of this term.

Reply:
I don't think we should be capitalizing in the main docs either. It looks like the UI uses lowercase, except for instances where it's only being capitalized as the label or title. 

<img width="578" alt="image" src="https://github.com/elastic/security-docs/assets/87339667/7052d5b1-83ea-44da-97cc-65d434bdf468">

<img width="631" alt="image" src="https://github.com/elastic/security-docs/assets/87339667/14ddab59-952e-40fb-a337-27dcfbc78a85">

Reply:
Oh good to know - I didn't even see that change come through. Thanks, I'll accept this edit and can open a doc issue if we need to correct it @benironside.

Reply:
Oh, the [source file](https://github.com/elastic/security-docs/blob/main/docs/getting-started/data-views-in-sec.asciidoc?plain=1) is using attributes for the name "data view" and it looks like they're the correct attributes — but they're not rendering correctly for some reason. #ThanksAsciiDoc